### PR TITLE
Magento_Integration: avoid using deprecated escape* methods from Abst…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

File: app/code/Magento/Integration/view/adminhtml/templates/integration/activate/permissions.phtml [hunk 1]
```diff
@@ -6,9 +6,14 @@
  * See COPYING.txt for license details.
  *
  * @var \Magento\Backend\Block\Widget\Form\Container $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
+
+/**
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Integration/view/adminhtml/templates/integration/activate/permissions.phtml]:
```suggestion
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Integration/view/adminhtml/templates/integration/activate/permissions/tab/webapi.phtml [hunk 2]
```diff
@@ -6,13 +6,17 @@
  * See COPYING.txt for license details.
  *
  * @var \Magento\Integration\Block\Adminhtml\Integration\Activate\Permissions\Tab\Webapi $block
+ * @var \Magento\Framework\Escaper $escaper
  */
 
-/** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
+/**
+ * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Integration/view/adminhtml/templates/integration/activate/permissions/tab/webapi.phtml]: here also duplicated escaper

File: app/code/Magento/Integration/view/adminhtml/templates/integration/popup_container.phtml [hunk 3]
```diff
@@ -6,9 +6,13 @@
  * See COPYING.txt for license details.
  *
  * @var \Magento\Backend\Block\Template $block
+ * @var \Magento\Framework\Escaper $escaper
  */
 
-/** @var \Magento\Backend\Block\Template $block */
+/**
+ * @var \Magento\Backend\Block\Template $block
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Integration/view/adminhtml/templates/integration/popup_container.phtml]: and here

File: app/code/Magento/Integration/view/adminhtml/templates/integration/tokens_exchange.phtml [hunk 4]
```diff
@@ -6,6 +6,11 @@
  * See COPYING.txt for license details.
  *
  * @var \Magento\Backend\Block\Template $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
+
+/**
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Integration/view/adminhtml/templates/integration/tokens_exchange.phtml]: and here

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Integration/view/adminhtml/templates/integration/activate/permissions.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Integration/view/adminhtml/templates/integration/activate/permissions.phtml]: fixed

File: app/code/Magento/Integration/view/adminhtml/templates/integration/activate/permissions/tab/webapi.phtml (hunk 2 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Integration/view/adminhtml/templates/integration/activate/permissions/tab/webapi.phtml]: fixed

File: app/code/Magento/Integration/view/adminhtml/templates/integration/popup_container.phtml (hunk 3 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Integration/view/adminhtml/templates/integration/popup_container.phtml]: fixed

File: app/code/Magento/Integration/view/adminhtml/templates/integration/tokens_exchange.phtml (hunk 4 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Integration/view/adminhtml/templates/integration/tokens_exchange.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, WebAPI Tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B

**Pull request closed without merging**